lib/FastBill/Api/BadRequestException.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace FastBill\Api;
+
+class BadRequestException extends \RuntimeException {
+
+    protected $errors = array();
+
+    public static function fromResponse(\stdClass $jsonResponse)
+    {
```

Comment:
indent with 2 spaces. use k&r style for braces (curly brace on same line as function statement)